databricks/koalas/frame.py

```diff
@@ -7090,19 +7090,22 @@ def isin(self, values) -> "DataFrame":
         if isinstance(values, dict):
             for i, col in enumerate(self.columns):
                 if col in values:
+                    item = values[col]
+                    item = item.tolist() if isinstance(item, np.ndarray) else list(item)
```

Comment:
Thanks for the comment :D

IIRC, `list()` itself calls the `__iter__` internally, so I believe `list(a)` and `list(iter(a))` exactly same.

And I think we still need `tolist()` since https://github.com/databricks/koalas/pull/2103#discussion_r594781305 ?

```python
>>> a = np.array([1,2,3])
>>> list(iter(a))
[1, 2, 3]
>>> type(list(iter(a))[0])
<class 'numpy.int64'>  # this should be `int`
>>> type(a.tolist()[0])
<class 'int'>
```

Please let me know if there is anything I missing!

Reply:
You are right, I think your method is correct.